src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -106,8 +52,8 @@ const ReactErrorUtils = {
     d: D,
     e: E,
     f: F,
-  ): Error | null {
-    return invokeGuardedCallback.apply(this, arguments);
+  ): void {
+    invokeGuardedCallback.apply(ReactErrorUtils, arguments);
```

Comment:
Just to make sure

```js
var {invokeGuardedCallback} = require('ReactErrorUtils');
```

still works (though I avoid `this` in `invokeGuardedCallback` anyway)